src/services/ParticipantProjectService.ts

```diff
@@ -0,0 +1,61 @@
+import { Response2 } from 'src/services/HttpService';
+import { ParticipantProject } from 'src/types/ParticipantProject';
```

Comment:
There's another version of `ParticipantProject` under `src/types/Participant`.. which will probably go away once we sync with BE types. Just a heads-up in case there's confusion later.